includes/class-simple-local-avatars.php

```diff
@@ -401,15 +401,19 @@ class="button item-delete submitdelete deletion"
 				<td colspan="2">
 					<fieldset id="simple-local-avatar-ratings" <?php disabled( empty( $profileuser->simple_local_avatar ) ); ?>>
 						<legend class="screen-reader-text"><span><?php esc_html_e( 'Rating' ); ?></span></legend>
-					<?php
-					if ( empty( $profileuser->simple_local_avatar_rating ) || ! array_key_exists( $profileuser->simple_local_avatar_rating, $this->avatar_ratings ) ) {
-						$profileuser->simple_local_avatar_rating = 'G';
-					}
+						<?php
+						$this->update_avatar_ratings();
 
-					foreach ( $this->avatar_ratings as $key => $rating ) :
-						echo "\n\t<label><input type='radio' name='simple_local_avatar_rating' value='" . esc_attr( $key ) . "' " . checked( $profileuser->simple_local_avatar_rating, $key, false ) . "/> $rating</label><br />";
-					endforeach;
-					?>
+						if ( empty( $profileuser->simple_local_avatar_rating ) || ! array_key_exists( $profileuser->simple_local_avatar_rating,
+								$this->avatar_ratings ) ) {
+							$profileuser->simple_local_avatar_rating = 'G';
+						}
+
+						foreach ( $this->avatar_ratings as $key => $rating ) :
+							echo "\n\t<label><input type='radio' name='simple_local_avatar_rating' value='" . esc_attr( $key ) . "' " . checked( $profileuser->simple_local_avatar_rating,
+									$key, false ) . "/>" . esc_html( $rating ) . "</label><br />";
```

Comment:
413 | ERROR | [x] Opening parenthesis of a multi-line function call must be the last content on the line
414 | ERROR | [x] Closing parenthesis of a multi-line function call must be on a line by itself